motion/guidance_interface/src/guidance_interface.py

```diff
@@ -216,6 +227,14 @@ def joystick_cb(self, msg):
         depthhold = buttons['B']  
         depthheadinghold = buttons['X']
         openloop = buttons['Y']
+        LB = buttons['LB']
+        RB = buttons['RB']
+        BACK = buttons['back']
+        SWITCH = buttons['start']
```

Comment:
Why use switch as the button? I understand it is used to switch between ROV and AUV-state, but wouldn't it be better to label it as start instead - since we are talking about the button

Reply:
I agree :) changed it